openxla-nvgpu/runtime/src/openxla/runtime/nvgpu/status_util.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef OPENXLA_RUNTIME_NVGPU_STATUS_UTIL_H_
+#define OPENXLA_RUNTIME_NVGPU_STATUS_UTIL_H_
+
+#include <stdint.h>
+
+#include "iree/base/api.h"
+#include "openxla/runtime/nvgpu/dynamic_symbols.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+// Converts a cudnnStatus_t to an iree_status_t.
+//
+// Usage:
+//   iree_status_t status = CUDNN_STATUS_TO_STATUS(cuDnnDoThing(...));
+#define CUDNN_STATUS_TO_STATUS(syms, expr, ...) \
```

Comment:
Is the `, ...` intentional? Seems dangerous to silently drop it.

Reply:
Not sure, it's copied/updated from CUDA HAL: https://github.com/openxla/iree/blob/main/runtime/src/iree/hal/drivers/cuda/status_util.h#L23